### ISSUE TITLE
Tagcloud format: the align attribute on the div element is obsolete.

### ISSUE THREAD

**Issue by jaideraf** [SemanticMediaWiki/SemanticResultFormats#253]
### Setup

- MW version: 1.28.2
- DB (MySQL): 5.6.36-82.0
- PHP version: 7.0.20 
- SMW version: 2.5.2
- SRF version: 3.0.0-alpha

### Issue

Just a minor issue, from the W3C Validator, showing that `tagcloud` format is using the `align` attribute on div elements.

**Steps to reproduce**

[See the W3C Validator message](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.semantic-mediawiki.org%2Fwiki%2FDemo%3ATagcloud%2FTagcloud_using_the_sphere_widget): 

```
Error: The align attribute on the div element is obsolete. Use CSS instead.
<div class="srf-tagcloud-sphere" data-version="0.4.1" align="justify">
```

**Expected result:**
Maybe just removing it (it did not changed the presentation to me):
```
<div class="srf-tagcloud-sphere" data-version="0.4.1">
```

**Comment by jaideraf** [comment]
Aditionally Chrome says:

```
This page is using the deprecated ResourceLoader module "jquery.ui.position".
This page is using the deprecated ResourceLoader module "jquery.ui.widget".
This page is using the deprecated ResourceLoader module "jquery.ui.core". 
Please use "mediawiki.ui.button" or "oojs-ui" instead.
```
As the author, maybe @JeroenDeDauw is interested in this.

**Comment by kghbln** [comment]
Thanks for reporting.

**Comment by kghbln** [comment]
I am not sure what the point of `align="justify"` was anyways. That was supposed to stretch the content to the with of the parent container so every word is displayed in equal width.

@jaideraf  It this really something one would want for a tagcloud? I myself do not. Besides, I think you do not see a difference before and after since `align="justify"` is no longer supported by the browser anyways.

If someone wants justify her or his data she or he could add `text-align: justify;` to the "srf-tagcloud-sphere" class.

**Comment by jaideraf** [comment]
> @jaideraf It this really something one would want for a tagcloud? I myself do not. 

No. Me, too. If nothing changes, with or without `align="justify"`, it's better without it.

**Comment by kghbln** [comment]
@jaideraf Thanks for your feedback confirming my view. The fix will ship with SRF 2.5.1 most probably next week.
